lib/btrfs_test.pm

```diff
@@ -114,6 +117,55 @@ sub cron_mock_lastrun {
     assert_script_run 'ls -al /var/spool/cron/lastrun/cron.{hourly,daily,weekly,monthly}';
 }
 
+# recent versions of snapper support json output
+# otherwise fallback to old implementation of parsing
+sub get_last_snap_number {
+    if (!defined($old_snapper)) {
+        $old_snapper = script_run('snapper --jsonout list');
```

Comment:
You can add `--disable-used-space` here though

Reply:
Added